### DISCUSSION TITLE
Im on 126.0.6478.178 with kernver 3 whats the max downgrade i can go to

### DISCUSSION THREAD

**Discussion by Balistic123** [3kh0/ext-remover#1134] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Im on 126.0.6478.178 with kernver 3 whats the max downgrade i can go to? because i t says i can go to 120 but when i tried downgrading to 120 it says out of date recovery device.

**Comment by s0urce-c0de** [comment]
What version is the recovery image? The patch was introduced in `v120.0.6099.315`, you might be using a version like `120.0.6099.246`, which is earlier.

**Reply by Balistic123** [reply]
2 things can you send me a website for a recovery image of v120.0.6099.315 and I'm on a manged Chromebook does that change anything?

**Reply by s0urce-c0de** [reply]
no, but what board do you have? recovery images change per board.

**Reply by Balistic123** [reply]
rammus

**Reply by s0urce-c0de** [reply, marked as answer]
try <a href="https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_15662.115.0 _rammus_recovery_stable-channel_mp-v4.bin.zip">this</a>, otherwise try [this](https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_15699.72.0_rammus_recovery_stable-channel_mp-v4.bin.zip)

**Reply by Balistic123** [reply]
the first link wouldn't even download? but the second one did

**Reply by synapt0** [reply]
Are there any recovery images on that version for the Octopus channel?

**Reply by s0urce-c0de** [reply]
You can find them [here](https://chrome100.dev/board/octopus)

**Reply by synapt0** [reply]
Damn. None for v120.

**Comment by Balistic123** [comment]
@s0urce-c0de question? is there any way to open the tool bar from a accessibility feature?

**Reply by s0urce-c0de** [reply]
... are you trying skiovox? i have no idea, but i don't try to discover skiovox exploits. try https://github.com/3kh0/ext-remover/discussions/1069#discussioncomment-9446593, you have v121, so it should work.

**Reply by Balistic123** [reply]
i have v126

**Reply by Balistic123** [reply]
because I'm trying to downgrade or find a new skiovox because I would like to do it @s0urce-c0de

**Reply by s0urce-c0de** [reply]
please don't ping me all the time, but do your best! I can't help you with skoivox (i don't have a chromebookto test on, my school takes them away on summer break)

**Comment by leoisbadatmakeingthings** [comment]
Try [this version](https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_15662.88.0_rammus_recovery_stable-channel_mp-v4.bin.zip), I've used skiovox on it before and downgraded to it with kernver 3.
I used [this skiovox method](https://www.youtube.com/watch?v=sD2E5mOeD-4). Should work if you can do alt+shift+s and esc.
